frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -102,10 +102,10 @@ export const createDeploymentConfig = (
     },
     spec: {
       replicas,
-      selector: podLabels,
+      selector: { ...labels, ...podLabels },
```

Comment:
Do not change the `selector`

Reply:
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#selector
It says here that `spec.selector` must match the `spec.template.metadata.labels`.

Reply:
I think `must match` is interpreted as the `selector` must correctly resolve based on the labels set at `spec.template.metadata.labels`. Not that they must be identical values.